Gopkg.lock

```diff
@@ -612,6 +612,14 @@
     "github.com/tektoncd/pipeline/pkg/client/clientset/versioned",
     "github.com/tektoncd/pipeline/pkg/client/clientset/versioned/fake",
     "github.com/tektoncd/pipeline/pkg/client/informers/externalversions",
+<<<<<<< HEAD
```

Comment:
The conflict is here :sweat_smile: Remove the conflict marker and re-run `dep ensure`, it should fix itself :wink: